### ISSUE TITLE
Support vibrancy effect

### ISSUE THREAD

**Issue by MaxHasADHD** [jerkoch/SwipeCellKit#10]
It would be cool to have a vibrant color background for view controllers presented over the current context and using a blur effect as the background. Standard UITableViewRowActions "support" it but currently broken and shows no text.

**Comment by jerkoch** [comment]
Thanks @MaxHasADHD ! 👍

Good catch... I totally missed it in `UITableViewRowAction`.  I'll try to add it when I have some.

**Comment by jerkoch** [comment]
Hi @MaxHasADHD - I took a stab at this today and got it all wired up.  But when I started testing, I realized the buttons actually overlap each other as they extend off the side of the view.  So I think we'll end up with effect views on top of effect views with side-by-side buttons. 

I can push my branch if you want to play around with it.

**Comment by MaxHasADHD** [comment]
@jerkoch Sure!

**Comment by jerkoch** [comment]
Hey @MaxHasADHD - Please take a look at the [visual-effect](/jerkoch/SwipeCellKit/tree/visual-effect) branch.  I also updated the example app to demo this feature.

A couple things to note:
* Button tap highlighting doesn't work since it currently darkens the background color of the button. The background color is clear when using backgroundEffect
* `SwipeTableOptions` needs `backgroundColor` set to `.clear` in order to hide the swipe menu background

I'm interested to hear your thoughts!

**Comment by MaxHasADHD** [comment]
I've been busy, but will hopefully look at it tonight! Thanks

**Comment by jerkoch** [comment]
Hey @MaxHasADHD - No worries... After I pushed the branch, I realized that I didn't really understand the background effect/vibrancy stuff, so the example app is a bit wonky!  😜  

I'm working on a new implementation where I think setting the background effect on the SwipeActionsView (the buttons container) in `SwipeTableOptions` makes more sense than doing it on each individual action.  I'll let you know how it goes!
